index.html

```diff
@@ -4,7 +4,64 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <title>setup-mb-skeleton</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>My logo</h1>
+      <img src="images/Menu.svg" alt="hamburger" />
+    </header>
+    <main class="wrapper">
+      <div class="banner">
+        <img src="images/Ellipse5.svg" alt="" />
+        <img src="images/Ellipse15.svg" alt="" class="top-circ icone-btn" />
+        <img src="images/Rectangle7.svg" alt="" />
+        <img src="images/Ellipse11.svg" alt="" class="aside-elip icone-btn" />
+      </div>
```

Comment:
- [ ] This image is not a part of the content, therefore please make sure to load it as a background image from `css`. You can export it from figma 😉. Or feel free to download the background image following the links: [mobile-bg-png](https://user-images.githubusercontent.com/77038610/158802622-3c68dfa5-f389-47b6-b860-ca6fee11c235.png) [mobile-bg-svg](https://user-images.githubusercontent.com/77038610/158802890-400e7761-c54f-4953-9958-78ad66290445.svg)

Reply:
Thank you for this image! I couldn't find how to download it in Figma.
let me place it real kick.

Reply:
Welcome 🐵